stackslib/src/net/stackerdb/mod.rs

```diff
@@ -148,6 +148,8 @@ pub const STACKERDB_INV_MAX: u32 = 4096;
 pub const STACKERDB_PAGE_LIST_MAX: u32 = 4096;
 /// maximum number of pages that can be used in a StackerDB contract
 pub const STACKERDB_MAX_PAGE_COUNT: u32 = 2;
+/// CHUNK_SIZE constant for signers StackerDBs
+pub const SIGNERS_STACKERDB_CHUNK_SIZE: u32 = 2 * 1024 * 1024; // 2MB
```

Comment:
This belongs in `libsigner`.  `libstackerdb` has a 16MB max

Reply:
Also, you can (and should) make this a `usize` so you can avoid a risky cast to `usize`.

Reply:
I've moved this specific max chunk size to `libsigner`. 

I haven't changed the existing constant in that file, because this const is really for limiting reads of chunk sizes for the signers contract, which has a lower max chunk size than other StackerDBs